org.osgi.test.common/src/test/java/org/osgi/test/common/dictionary/DictionariesTestCase.java

```diff
@@ -67,10 +80,24 @@ public void setUp() throws Exception {
 		map.put("key4", "value4");
 		map.put("key5", "value5");
 
+		sr = mock(ServiceReference.class);
```

Comment:
I'm a big fan of Mockito, but its use does come at a price. Given that you're not actually using any of Mockito's `verify()` methods, I suspect in this case it might have been easier (and more readable/maintainable in the longer term) to directly create in source a "fake" implementation of `ServiceReference`, rather than dynamically create one using `mock()` and `when()`. I'd leave it as is for now though, more of a note for future reference.

Reply:
In my first approach i had a `FakeServiceReference implements ServiceReference`. But that indicated bnd to add a
`Import-Package: org. osgi.framework;version="[1.8,1.9) ` but our eclipse hast an framework package version is `>1.9`

Reply:
You could override the Import-Package to make it `[1.8,2.0)`. I'm actually surprised that Bnd didn't do this by default?